cerebrium/endpoints/async.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: "Asynchronous Execution (Preview)"
+description: "Execute calls to a Cerebrium app to be run asynchroously"
+---
+
+<Note>
+This is a new feature! As such, the API is still currently subject to changes.
+
+Most applications are expected to work with the current implementation.
+However, should you encounter an issue deploying a Custom Runtime please
+reach out to us on Discord!
+
+</Note>
+
+Many applications require asynchronous execution of functions in a "fire-and-forget" fashion
+in order to work optimally. In this scenario, you hand off execution of a function to be Cerebrium's
+responsibility, while you as the developer are responsible for ensuring that data is able leave the function.
+
+Through parameterizing your request to the Cerebrium API, you can enable your function to executed asynchronously
+by setting the `async` query parameter of your request to `true`. This would look something like this:
+
+```bash
+curl -X POST https://api.cortex.cerebrium.ai/v4/<YOUR-PROJECT-ID>/<YOUR-APP>/run?async=true'\
+       -H 'Content-Type: application/json'\
+       -H 'Authorization: Bearer <YOUR-JWT-TOKEN>\
+       --data '{"param": "hello world"}'
+```
+
+This would _immediately_ return a response akin to the following, with a body that only specifies a `run_id`:
```

Comment:
Maybe add:
- This run id can be used to associate your workloads to requests.
- In future we will add the ability to check the status of requests + cancel requests